flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Engine.js

```diff
@@ -69,11 +73,22 @@ var A320_Neo_LowerECAM_Engine;
             let n2Igniting = (SimVar.GetSimVarValue("TURB ENG IS IGNITING:1", "bool") === 1) ? true : false;
             let n2Percent = SimVar.GetSimVarValue("ENG N2 RPM:1", "percent");
             if (engineStarting && n2Igniting && n2Percent > 18 && n2Percent < 55) {
-                if (this.ignRightCurrentState == A320_Neo_LowerECAM_Engine.Definitions.IGN_STATE.A) {
-                    ignLeftTargetState = A320_Neo_LowerECAM_Engine.Definitions.IGN_STATE.B;
+                if (this.ignLeftOn != true) {
+                    if (this.ignLeftLastUsedIgniter == A320_Neo_LowerECAM_Engine.Definitions.IGN_STATE.NONE) {
```

Comment:
**suggestion**: Use `===` as you don't intent to do any type-coercion here.

Reply:
Done

Reply:
There are some other places where e.g. `!=` is used which should/can be replaced with `!==`.

Reply:
Done :)